_data/simple-icons.json

```diff
@@ -1568,6 +1568,12 @@
             "hex": "54B848",
             "source": "https://www.checkmarx.com/resources/datasheets/"
         },
+        {
+            "title": "Checkmk",
+            "hex": "13D389",
+            "source": "https://github.com/animesh-anand/Checkmk-Logo/blob/main/checkmk.svg",
```

Comment:
Since they provide official versions for their assets, it would be preferable to use it for the source:
https://checkmk.com/company/brand-assets